parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -193,6 +193,21 @@ write_files:
 {{end}}
 
 {{if NeedsContainerd}}
+- path: /etc/sysctl.d/11-containerd.conf
+  permissions: "0644"
+  owner: root
+  content: |
+    net.ipv4.ip_forward = 1
+    #EOF
+
+- path: /etc/systemd/system/containerd.service.d/exec_start.conf
+  permissions: "0644"
+  owner: root
+  content: |
+    [Service]
+    ExecStartPre=/sbin/iptables -P FORWARD ACCEPT
```

Comment:
WDYT if instead of changing the default forwarding policy if we do something like:

```
-A FORWARD -i azure0 -o azure0 -j ACCEPT
-A FORWARD -i azure0 ! -o azure0 -j ACCEPT
```

This way we only forward packets for azure0.

Reply:
Yes, let's do this as a follow-up for both docker and containerd (as far as I can tell this additional targeted hardening is appropriate for any CRI).